QUEST_LV_0200.tsv

```diff
@@ -4890,21 +4890,21 @@ QUEST_LV_0200_20150323_004888	Talk to the disciple, Ruodell
 QUEST_LV_0200_20150323_004891	You've found the documents that would help removing the curse of Naktis. Talk to the disciple, Ruodell.
-QUEST_LV_0200_20150323_004892	Talk to the disciple, Yosana
-QUEST_LV_0200_20150323_004893	The disciple, Yosana seems that she doesn't know what to do. Talk to the disciple, Yosana again.
-QUEST_LV_0200_20150323_004894	Retrieve the research documents of the disciple, Yosana
-QUEST_LV_0200_20150323_004895	The disciple, Yosana is telling you that the research documents have disappeared and she is suspecting the demons. Defeat the demons and retrieve her research documents.
-QUEST_LV_0200_20150323_004896	You've found all the research documents of the disciple, Yosana. Hand them over to the disciple, Yosana.
-QUEST_LV_0200_20150323_004897	Retrieve %s
-QUEST_LV_0200_20150323_004898	Talk to the disciple, Daram
-QUEST_LV_0200_20150323_004899	Disciple Daram needs your help. Talk to her.
-QUEST_LV_0200_20150323_004900	Collect the solvents using the scroll of purification to the demons
-QUEST_LV_0200_20150323_004901	Disciple Daram told you that many solvents are needed to make the reagent. Use the scroll of purification on the demons and collect the solvents.
-QUEST_LV_0200_20150323_004902	You've collected enough murky solvents. Hand them over to Disciple Daram.
-QUEST_LV_0200_20150323_004903	Disciple Daram is thinking about something. Talk to her.
-QUEST_LV_0200_20150323_004904	Use with the reagent Daram created on the demons
-QUEST_LV_0200_20150323_004905	Daram told you that she successfully created the reagent, but she needs to experiment with it. Experiment the medicine on the demons.
-QUEST_LV_0200_20150323_004906	The reagent caused various effects. Tell Daram about the effects of the reagent.
+QUEST_LV_0200_20150323_004892	$Talk to Priest Yosana
+QUEST_LV_0200_20150323_004893	$Priest Yosana seems to be at a loss. Talk to Priest Yosana again.
+QUEST_LV_0200_20150323_004894	$Retrieve Priest Yosana's Research
+QUEST_LV_0200_20150323_004895	$Priest Yosana told you that her research documents have disappeared and she suspects the demons have a role. Defeat the demons and retrieve her research documents.
+QUEST_LV_0200_20150323_004896	$You've found all of Priest Yosana's documents. Hand them over to Priest Yosana.
+QUEST_LV_0200_20150323_004897	$Retrieve %s
+QUEST_LV_0200_20150323_004898	$Talk to Priest Daram
+QUEST_LV_0200_20150323_004899	$Priest Daram needs your help. Talk to her.
+QUEST_LV_0200_20150323_004900	$Collect solvents by using the Scroll of Purification on the demons
+QUEST_LV_0200_20150323_004901	$Priest Daram told you that many solvents are needed to make the reagent. Use the scroll of purification on the demons and collect the solvents.
+QUEST_LV_0200_20150323_004902	$You've collected enough turbid solvents. Hand them over to Priest Daram.
+QUEST_LV_0200_20150323_004903	$Priest Daram is thinking about something. Talk to her.
+QUEST_LV_0200_20150323_004904	$Use with the reagent Daram created on the demons
```

Comment:
It was like that in your PR. I just added 'Priest' to it. Anyway, I will add your edit in my PR.